filebeat/harvester/log_file.go

```diff
@@ -156,8 +161,15 @@ func (r *LogFile) errorChecks(err error) error {
 }
 
 func (r *LogFile) wait() {
+
 	// Wait before trying to read file wr.ch reached EOF again
-	time.Sleep(r.backoff)
+	timer := time.NewTimer(r.backoff)
```

Comment:
Yes, but other PR. Already on the list of https://github.com/elastic/beats/issues/1913